### PR TITLE
Add support for Rails 5

### PR DESCRIPTION
**Pull request by sinsoku**
see also:
  - https://github.com/rails/rails/commit/3806eb70a63f0252c88ebb25f0aa79b4c0ca5686
  - https://github.com/rails/rails/commit/78a5124bf09d0cd285d4d1a6242bd67badb2f621

### CODE REVIEW

File: gemfiles/rails_5_beta4.gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,164 @@
+PATH
```
Comment (mizoR) [inline gemfiles/rails_5_beta4.gemfile.lock]: Cannot you remove this lock file?

File: lib/active_decorator/rspec.rb [hunk 2]
```diff
@@ -15,7 +15,11 @@ def self.enable(example)
                      ActionController::Base
                    end
       controller = Class.new(base_class).new
-      controller.request = ActionController::TestRequest.new
+      controller.request = if ActionPack::VERSION::STRING >= '5'
```
Comment (mizoR) [inline lib/active_decorator/rspec.rb]: I prefer `ActionController::TestRequest.respond_to?(:create)` to `ActionPack::VERSION::STRING >= '5' `.  But how about you?

File: gemfiles/rails_5_beta4.gemfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
```
Comment (mizoR) [inline gemfiles/rails_5_beta4.gemfile]: Could you rename this to `gemfiles/Gemfile-rails.5.0.0.beta4`. This is my taste 😅

File: .travis.yml [hunk 4]
```diff
@@ -6,6 +6,17 @@ rvm:
   - 2.2.4
   - 2.3.0
 
+gemfile:
+  - Gemfile
+  - gemfiles/Gemfile-rails.5.0.0.beta4.gemfile
```
Comment (mizoR) [inline .travis.yml]: `s/gemfiles/Gemfile-rails.5.0.0.beta4.gemfile/gemfiles/Gemfile-rails.5.0.0.beta4`

**Comment by sinsoku** [comment]
Thanks for feedback. I done.

**Comment by mizoR** [comment]
Thank you for your contribute, I released [v0.0.8](https://rubygems.org/gems/active_decorator-rspec/versions/0.0.8) 😄

**Pull request merged**